src/Widgets/PanelLabel.vala

```diff
@@ -39,7 +39,7 @@ public class DateTime.Widgets.PanelLabel : Gtk.Grid {
         add (date_revealer);
         add (time_label);
 
-        var clock_settings = new GLib.Settings ("org.gnome.desktop.interface");
+        var clock_settings = new GLib.Settings ("io.elementary.desktop.wingpanel.datetime");
         clock_settings.bind ("clock-format", this, "clock-format", SettingsBindFlags.DEFAULT);
```

Comment:
clock_setting name should be "clock-formats" to match schema.

Reply:
The schema is `clock-format` it's only the enum that is `-formats`

![screenshot from 2018-08-08 12 54 41 2x](https://user-images.githubusercontent.com/7277719/43860965-3d427c2e-9b0a-11e8-9ad9-e85a2da9ec8c.png)